padsearch/padsearch.py

```diff
@@ -655,7 +655,8 @@ def __init__(self, bot):
                 # If it still failed, raise the original exception
                 raise ex
         dg_cog = self.bot.get_cog('Dadguide')
-        monsters = dg_cog.database.all_monsters()
+        await dg_cog.wait_until_ready()
```

Comment:
I don't think you should include this. `wait_until_ready()` is intended for use by cogs during startup, not normal operation. I'd rather this fails fast.

Other than that changes LGTM